paymentmethod.go

```diff
@@ -2,6 +2,14 @@ package stripe
 
 import "encoding/json"
 
+// List of values that FPX AccountHolderType can take.
+type AccountHolderType string
```

Comment:
It shouldn't be named `AccountHolderType` here. It should be `PaymentMethodFPXAccountHolderType` instead

Reply:
Done, thanks.